app/models/miq_ae_datastore.rb

```diff
@@ -24,6 +24,10 @@ def self.temp_domain
     "#{TEMP_DOMAIN_PREFIX}-#{SecureRandom.uuid}"
   end
 
+  def self.datastore_directory
```

Comment:
Not a fan of wrapping a method around a constant.  Particularly because we probably don't want this to be a public method, but this seems like the only issue.